package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/icaraps"
   },
   "dependencies": {
-    "@adobe/gatsby-theme-aio": "4.9.12",
+    "@adobe/gatsby-theme-aio": "^4.13.0",
```

Comment:
is this change necessary ?

Reply:
Yeah reverted and checked. Deployment is successful without this change as well.